src/docker/era_5g_netapp_base_mmcv/Dockerfile.gpu

```diff
@@ -0,0 +1,29 @@
+FROM nvidia/cuda:11.7.1-base-ubuntu20.04
```

Comment:
Update to [11.8.0-base-ubuntu20.04](https://hub.docker.com/layers/nvidia/cuda/11.8.0-base-ubuntu20.04/images/sha256-436324fa946943e170fb246d8e911241c0b93bd5fb6256a964a7d01b06dfcf68?context=explore) or [12.0.1-base-ubuntu20.04](https://hub.docker.com/layers/nvidia/cuda/12.0.1-base-ubuntu20.04/images/sha256-f1a664ad635f1aff63afcad0cc62f122dede7bb350106857b6faebe286aeb1e4?context=explore) (if we don't need to stick to version 11 for some reason)?

Reply:
We may try it with 11.8. According to https://github.com/open-mmlab/mmdetection/blob/master/docs/en/get_started.md/#Installation 12 is not compatible.

Reply:
Ok, according to https://pytorch.org/get-started/locally/ the current stable version of PyTorch requires the CUDA 11.7, so we can not use 11.8 at the moment.

Reply:
Comment added